skimage/_shared/filters.py

```diff
@@ -61,16 +100,12 @@ def gaussian(
 
         .. versionadded:: 0.19
            ``channel_axis`` was added in 0.19.
+    out : ndarray, optional
+        The ``out`` parameter passes an array in which to store the
```

Comment:
```suggestion
        The `out` parameter passes an array in which to store the
```
https://github.com/scikit-image/scikit-image/pull/7154#discussion_r1374763766 :sweat_smile: 

Also, I would reword more lightly: "The `out` parameter is for storing the filtered array (output)."